common/svc/beep.c

```diff
@@ -104,3 +122,20 @@ void svc_beep_hour_set_freq(uint16_t f) {
 void svc_beep_hour_set_duration(uint16_t d) {
 	beep_hour_duration = d;
 }
+
+uint8_t svc_beep_hour_quiet_get_enable(void) {
+	return beep_hour_quiet_enable;
+}
+
+void svc_beep_hour_quiet_set_enable(uint8_t e) {
+	beep_hour_quiet_enable = !!e;
+}
+
+uint16_t svc_beep_hour_quiet_get_interval(void) {
+	return ((beep_hour_quiet[0] * 100) + beep_hour_quiet[1]);
+}
```

Comment:
I'm not exactly happy with this one: Turning otherwise unrelated numbers (like hours and minutes) into a "string" number (minutes+hours*100) is an oddity of `svc_menu_item_adj_t` and shouldn't spill into other layers of the firmware. My suggestion: `void svc_beep_hour_quiet_get_interval(uint8_t *s, uint8_t *e)`